compiler/rustc_typeck/src/expr_use_visitor.rs

```diff
@@ -588,6 +588,7 @@ fn copy_or_move<'a, 'tcx>(
     mc: &mc::MemCategorizationContext<'a, 'tcx>,
     place_with_id: &PlaceWithHirId<'tcx>,
 ) -> ConsumeMode {
+    // FIXME(eddyb) add `|| type_is_must_clone(ty)`.
```

Comment:
Oh I need to fix this so we don't end up with by-ref captures that the MIR borrowck later errors on because we generate MIR `Move`s out of those.